src/data_utils/dataset_EDA/test_data_utilities_find_outlier_qrcodes.py

```diff
@@ -27,35 +27,35 @@ def prepare_test_df():
 
 def test_find_outlier_qrcodes_age_min():
     df = prepare_test_df()
-    qrs = find_outlier_qrcodes(df, 'age', '<365/2')
+    qrs = find_outliers(df, 'age', '<365/2', 'qr')
```

Comment:
Consider re-writing to 

```suggestion
    qrs = find_outliers(df, 'age', '<365/2', data_id_name='qr')
```

in order to improve readability